src/components-examples/material/datepicker/date-range-picker-comparison/date-range-picker-comparison-example.html

```diff
@@ -1,5 +1,6 @@
 <mat-form-field class="example-form-field" appearance="fill">
   <mat-label>First campaign</mat-label>
+  <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
```

Comment:
I'm wondering if users would actually infer that this is a date format. I would assume that the screen reader will read it out as an abbreviation.

Reply:
This is what is looks like visually.

<img width="309" alt="image" src="https://user-images.githubusercontent.com/7720245/160730642-a8c856af-40c7-4dbd-95be-8dd804ae8d69.png">
<img width="318" alt="image" src="https://user-images.githubusercontent.com/7720245/160730654-cbf1912b-372d-4b07-a4a3-dccdfca006b9.png">

The hint visually matches the input above it. I tested this on VoiceOver and it announced this as an abbreviation.

Reply:
That's what I meant. Would users actually know that the abbreviation refers to the date format?

Reply:
Hmmm. Showing the date format as an abbreviation is fairly conventional AFAIK.

As far as screen readers as concerned, I would recommend not dictating exactly how it is announced. People get used to how screen readers work and trying to control too much what the screenreader says can make it worse.

Reply:
This seems standard to me as well, I see many websites showing hints like this

Reply:
FWIW, we have another component internally that uses abbreviations.